dm/pkg/conn/basedb.go

```diff
@@ -69,16 +69,14 @@ func (d *DefaultDBProviderImpl) Apply(config *config.DBConfig) (*BaseDB, error)
 		if loadErr := config.Security.LoadTLSContent(); loadErr != nil {
 			return nil, terror.ErrCtlLoadTLSCfg.Delegate(loadErr)
 		}
-		tlsConfig, err := util.ToTLSConfigWithVerifyByRawbytes(config.Security.SSLCABytes,
-			config.Security.SSLCertBytes, config.Security.SSLKEYBytes, config.Security.CertAllowedCN)
+		tlsConfig, err := util.NewTLSConfig(
```

Comment:
I find `InsecureSkipVerify` is always true from API `util.NewTLSConfig`, is this as expected?

Reply:
https://github.com/pingcap/tidb/pull/37479#discussion_r962687063

we use `VerifyPeerCertificate` to do explicit check